machine_learning/forecasting/run.py

```diff
@@ -0,0 +1,158 @@
+    prediction = abs(beta[0] + test_dt[0] * beta[1] + test_mtch[0] + beta[2])
+    return prediction
+
+
+def sarimax_predictor(train_user: list, train_match: list, test_match: list) -> float:
+    """
+    second method: Sarimax
+    sarimax is a statistic method which using previous input
+    and learn its pattern to predict future data
+    input : training data (total_user, with exog data = total_event) in list of float
+    output : list of total user prediction in float
+    >>> sarimax_predictor([4,2,6,8], [3,1,2,4], [2])
+    6.6666671111109626
+    """
+    order = (1, 2, 1)
+    seasonal_order = (1, 1, 0, 7)
+    model = SARIMAX(
+        train_user, exog=train_match, order=order, seasonal_order=seasonal_order
+    )
+    model_fit = model.fit(disp=False, maxiter=600, method="nm")
+    result = model_fit.predict(1, len(test_match), exog=[test_match])
+    return result[0]
+
+
+def support_machine_regressor(x_train: list, x_test: list, train_user: list) -> float:
+    """
+    Third method: Support vector regressor
+    svr is quite the same with svm(support vector machine)
+    it uses the same principles as the SVM for classification,
+    with only a few minor differences and the only different is that
+    it suits better for regression purpose
+    input : training data (date, total_user, total_event) in list of float
+    where x = list of set (date and total event)
+    output : list of total user prediction in float
+    >>> support_machine_regressor([[5,2],[1,5],[6,2]], [[3,2]], [2,1,4])
+    1.634932078116079
+    """
+    regressor = SVR(kernel="rbf", C=1, gamma=0.1, epsilon=0.1)
+    regressor.fit(x_train, train_user)
+    y_pred = regressor.predict(x_test)
+    return y_pred[0]
+
+
+def interquartile_range_checker(train_user: list) -> float:
+    """
+    Optional method: interquatile range
+    input : list of total user in float
+    output : low limit of input in float
+    this method can be used to check whether some data is outlier or not
+    >>> interquartile_range_checker([1,2,3,4,5,6,7,8,9,10])
+    2.8
+    """
+    train_user.sort()
+    q1 = np.percentile(train_user, 25)
+    q3 = np.percentile(train_user, 75)
+    iqr = q3 - q1
+    low_lim = q1 - (iqr * 0.1)
+    return low_lim
+
+
+def data_safety_checker(list_vote, actual_result):
+    """
+    Used to review all the votes (list result prediction)
+    and compare it to the actual result.
+    input : list of predictions
+    output : print whether it's safe or not
+    >>> data_safety_checker([2,3,4],5)
+    today's data is not safe
+    """
+    safe = 0
+    not_safe = 0
+    for i in list_vote:
+        if i > actual_result:
+            safe = not_safe + 1
+        else:
+            if abs(abs(i) - abs(actual_result)) <= 0.1:
+                safe = safe + 1
+            else:
+                not_safe = not_safe + 1
+    if safe > not_safe:
+        print("today's data is safe")
+    else:
+        print("today's data is not safe")
```

Comment:
```suggestion
    print("today's data is {'not ' if safe <= not_safe else ''}safe")
```

Reply:
idk whether that code is working, but i got ur point to use inline if print

--done